src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs

```diff
@@ -532,11 +542,17 @@ private static FormulaValue DateImpl(EvalVisitor runner, EvalVisitorContext cont
             // such as: Date(2000, 25, 69) -> 3/10/2002
             try
             {
-                var datetime = new DateTime(year, 1, 1)
+                var timeZoneInfo = runner.TimeZoneInfo;
+
+                var dateTimeKind = timeZoneInfo.BaseUtcOffset == TimeSpan.Zero ?
```

Comment:
> timeZoneInfo.BaseUtcOffset == TimeSpan.Zero [](http://example.com/codeflow?start=34&length=44)

This expression is tricky -  can we factor behind a property? 

var dateTimeKind  = runner.GetTimeZoneKind;